src/briefcase/platforms/linux/system.py

```diff
@@ -247,6 +213,68 @@ def app_python_version_tag(self, app):
             python_version_tag = super().app_python_version_tag(app)
         return python_version_tag
 
+    def deb_abi(self, app: AppConfig) -> str:
+        """The default ABI for dpkg for the target environment."""
+        try:
+            return self._deb_abi
+        except AttributeError:
+            self._deb_abi = (
+                self.tools[app]
+                .app_context.check_output(["dpkg", "--print-architecture"])
+                .strip()
+            )
+            return self._deb_abi
+
+    def rpm_abi(self, app: AppConfig) -> str:
+        """The default ABI for rpm for the target environment."""
+        try:
+            return self._rpm_abi
+        except AttributeError:
+            self._rpm_abi = (
+                self.tools[app]
+                .app_context.check_output(["rpm", "--eval", "%_target_cpu"])
+                .strip()
+            )
+            return self._rpm_abi
```

Comment:
I don't think this solution actually works for 32bit RHEL on 64bit hardware....but 1) I can't actually get Fedora to work in this configuration and 2) I'm tired of reading `rpm` docs. This works fine for typical operation.